ng-dev/release/set-dist-tag/cli.ts

```diff
@@ -46,21 +55,36 @@ async function handler(args: Arguments<ReleaseSetDistTagOptions>) {
   if (version === null) {
     error(red(`Invalid version specified (${rawVersion}). Unable to set NPM dist tag.`));
     process.exit(1);
+  } else if (isExperimentalSemver(version)) {
+    error(
+      red(
+        `Unexpected experimental SemVer version specified. This command expects a ` +
+          `non-experimental project SemVer version.`,
+      ),
+    );
+    process.exit(1);
   }
 
   debug(`Setting "${tagName}" NPM dist tag for release packages to v${version}.`);
   const spinner = new Spinner('');
 
-  for (const pkgName of npmPackages) {
-    spinner.update(`Setting NPM dist tag for "${pkgName}"`);
+  for (const pkg of npmPackages) {
+    // If `--skip-experimental-packages` is specified, all NPM packages which
+    // are marked as experimental will not receive the NPM dist tag update.
+    if (pkg.experimental && skipExperimentalPackages) {
+      continue;
```

Comment:
I think we should log/update the spinner expressing that we skipped a package because its experimental

Reply:
It should rarely be readable anyway, but I'll add a message.